fluid-project/hearth
# feat: add organization, memberships and invitations (resolves #5)

This PR adds an `Organization` model which users can join via a `Membership` model, and which users can be invited to join via an `Invitation` model. It also adds feature test stubs for users, two-factor authentication, and organizations, memberships and invitations.

### Testing

1. Install this branch with Composer.
2. Run the Hearth installer using the `--organizations` flag: `php artisan hearth:install --organizations`
3. Log in and navigate to `/en/organizations/create`.
4. Verify that you can create an organization.
5. Verify that you can edit your organization.
6. Verify that you can invite a new user to the organization.
7. Verify that you can register and log in as the new user and then accept the invitation.
8. Verify that if an organization has only one administrator, they cannot be removed from the organization.
9. Verify that non-administrative members of an organization cannot:
	- remove other users from the organization;
	- invite new users to the organization;
	- delete the organization.
10. Verify that administrators of an organization can delete the organization.

### Additional Information

- Blocked by #22.
